exchanges/btse/btse_wrapper.go

```diff
@@ -412,22 +412,33 @@ func (b *BTSE) GetExchangeHistory(p currency.Pair, assetType asset.Item, timesta
 	var resp []exchange.TradeHistory
 	for x := range trades {
 		tempExch := exchange.TradeHistory{
-			Timestamp: trades[x].Time,
+			Timestamp: time.Unix(0, trades[x].Time*int64(time.Millisecond)),
 			Price:     trades[x].Price,
 			Amount:    trades[x].Amount,
 			Exchange:  b.Name,
+			Side:      trades[x].Side,
 			Type:      trades[x].Type,
 			TID:       strconv.Itoa(trades[x].SerialID),
 		}
+
 		resp = append(resp, tempExch)
 	}
 	return resp, nil
 }
 
-func (b *BTSE) withinLimits(pair string, amount float64) bool {
-	return (math.Mod(amount, b.OrderSizeLimits[pair].MinSizeIncrement) == 0) ||
-		amount < b.OrderSizeLimits[pair].MinOrderSize ||
-		amount > b.OrderSizeLimits[pair].MaxOrderSize
+func (b *BTSE) withinLimits(pair currency.Pair, a asset.Item, amount float64) bool {
+	fPair, err := b.FormatExchangeCurrency(pair, a)
+	if err != nil {
+		return false
+	}
```

Comment:
This might not be necessary as we already should call this in the wrapper function.